wazuhapp/generate_wazuh_app.sh

```diff
@@ -65,7 +67,7 @@ compute_version_revision(){
 
 download_sources(){
 
-    git clone https://github.com/wazuh/wazuh-kibana-app -b ${BRANCH_TAG} --depth=1 ${SOURCES_DIRECTORY}
+    git clone https://github.com/wazuh/wazuh-kibana-app -b ${BRANCH_TAG} --depth=1 ${SOURCES_DIRECTORY} || clean 1
```

Comment:
`clean` should be executed outside.

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/329/commits/9d5bae829939b1955df6e1faffb2ceffe6c7a8b5.